clients/drcachesim/drpt2trace/drpt2trace.cpp

```diff
@@ -261,6 +264,29 @@ option_init(int argc, const char *argv[])
             return false;
         }
     }
+
+    if (!op_elf.specified() && !op_primary_sb.specified()) {
+        std::cerr << CLIENT_NAME << ": at least one of the options --elf, --primary_sb."
+                  << std::endl;
+        print_usage();
+        return false;
+    }
+
+    if (op_primary_sb.specified()) {
+        std::vector<droption_parser_t *> sb_required_op_list;
+        sb_required_op_list.push_back(&op_sb_sample_type);
+        sb_required_op_list.push_back(&op_sb_time_zero);
+        sb_required_op_list.push_back(&op_sb_time_shift);
+        sb_required_op_list.push_back(&op_sb_time_mult);
+        for (auto &op : sb_required_op_list) {
```

Comment:
We check the required_ops only for the op_primary_sb case? We should do it for the op_elf case too right?